### DISCUSSION TITLE
How to set MicrometerOptions.histogram to false in SpringBoot?

### DISCUSSION THREAD

**Discussion by siddheshlatkar** [redis/lettuce#2600] [Q&A]
How to declaratively disable [MicrometerOptions.histogram](https://github.com/lettuce-io/lettuce-core/blob/main/src/main/java/io/lettuce/core/metrics/MicrometerOptions.java#L56)?

We want to set it to false to disable [lettuce.command.completion](https://github.com/lettuce-io/lettuce-core/blob/main/src/main/java/io/lettuce/core/metrics/MicrometerCommandLatencyRecorder.java#L119 ) and [lettuce.command.firstresponse ](https://github.com/lettuce-io/lettuce-core/blob/main/src/main/java/io/lettuce/core/metrics/MicrometerCommandLatencyRecorder.java#L135) metrics

**Comment by mp911de** [comment]
This is a question for the Spring Boot team as Lettuce isn't using Spring Boot. You can programmatically configure `MicrometerOptions` by registering it as bean.
